tests/Integration/AuthenticationTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace FtwSoft\Rundeck\Tests\Integration;
+
+use FtwSoft\Rundeck\Authentication\PasswordAuthentication;
+use FtwSoft\Rundeck\Authentication\TokenAuthentication;
+use FtwSoft\Rundeck\Client;
+use FtwSoft\Rundeck\Exception\AuthenticationException;
+use GuzzleHttp\Client as HttpClient;
+use Http\Factory\Guzzle\RequestFactory;
+use Http\Factory\Guzzle\StreamFactory;
+use PHPUnit\Framework\TestCase;
+
+class AuthenticationTest extends TestCase
+{
+    /**
+     * @var string
+     */
+    private $rundeckHost;
```

Comment:
type hints for tests are needed

Reply:
Property types are a php 7.4 feature. My changes would run under `^7.2 || ^8.0`. We would need to change that to `^7.4 || ^8.0`